lib/pages/Friendprofile_page.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:flutter/material.dart';
+class FriendProfileWidget extends StatefulWidget {
+  FriendProfileWidget({Key key}) : super(key: key);
+
+  @override
+  _FriendProfileWidgetState createState() => _FriendProfileWidgetState();
+}
+
+class _FriendProfileWidgetState extends State<FriendProfileWidget> {
+  final scaffoldKey = GlobalKey<ScaffoldState>();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      key: scaffoldKey,
+      backgroundColor: Colors.black,
+      floatingActionButton: FloatingActionButton(
+        onPressed: () {
+          print('FloatingActionButton pressed ...');
+        },
+        backgroundColor: Color(0xFFB39738),
```

Comment:
Can we capture this colors in the constant class

Reply:
Its done